core/arch/aarch64/instr_create.h

```diff
@@ -38,6 +38,10 @@
 #include "../instr_create_shared.h"
 #include "instr.h"
 
+#define FSZ_HALF 1
```

Comment:
These should be exported (i.e., below the DR_API EXPORT BEGIN) and they should have Doxygen comments: users need to know about these, right?

Reply:
Maybe there should be a prose explanation somewhere in the IR sections (or in a new section) in bt.dox as well?

Reply:
Done. I've added a section to the docs, but I am not really sure if it is at the right place and the title makes sense. It would be great if you could have a look. If we add this section, I think we should also explain the OP_xx opcode. And lots of the docs talk about IA-32, even though ARM and AArch64 are supported in most cases as well. I try to update the docs sometime.

Reply:
I added some comments.

I guess I thought OP_xx was temporary?  If there is a plan to use it for future opcodes where you know somehow that it's legal but don't know its name, I would vote for OP_UNKNOWN or something more explicit.